packages/back-end/src/services/experimentNotifications.ts

```diff
@@ -441,37 +411,92 @@ export const notifySignificance = async ({
   );
 };
 
+export const notifyDecision = async ({
+  context,
+  experiment,
+  currentStatus,
+  lastStatus,
+}: {
+  context: Context;
+  experiment: ExperimentInterface;
+  currentStatus: RunningExperimentStatusData;
+  lastStatus?: RunningExperimentStatusData;
+}) => {
+  if (
+    currentStatus.status === "ship-now" ||
+    currentStatus.status === "rollback-now" ||
+    currentStatus.status === "ready-for-review"
+  ) {
+    const eventType: "ship" | "rollback" | "review" = (() => {
+      switch (currentStatus.status) {
+        case "ship-now":
+          return "ship";
+        case "ready-for-review":
+          return "review";
+        case "rollback-now":
+          return "rollback";
+      }
+    })();
+
+    if (currentStatus.status !== lastStatus?.status) {
+      dispatchEvent({
```

Comment:
question: why are we not using `memoizeNotification` here?

Reply:
I kind of want this one to always fire if it differs from the last status, even if it has fired for the experiment before.